### PR TITLE
Localize runtime #2335

### PR DESCRIPTION
**Pull request by matteobisi**
### Describe your changes
localize Runtime IT

### Related issue number or link (ex: `resolves #issue-number`)
#2335 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/it/runtime.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Quali problematiche affronta
+Senza l'astrazione di un runtime del container, l'applicazione dovrebbe occuparsi di tutte le meccaniche di ciascun sistema operativo, aumentando la complessità dell'esecuzione dell'app.
+
+## In che modo aiuta
+I container runtime sono un componente necessario degli orchestratori di container come Kubernetes. 
+Gestiscono il ciclo di vita del container, che fa principalmente tre cose: 
+definisce come vengono specificate le immagini del container e come il runtime può recuperarle;
+gestiscono come queste immagini vengono decompresse, stratificate, montate ed eseguite;
+gestiscono le risorse hardware occupandosi di tutte queste azioni a livello di sistema operativo, come allocazione e isolamento delle risorse.
```
Comment (ugho16) [inline content/it/runtime.md:24]:
```suggestion

- definisce come vengono specificate le immagini del container e come il runtime può recuperarle;
- gestisce come queste immagini vengono decompresse, stratificate, montate ed eseguite;
- gestisce le risorse hardware occupandosi di tutte queste azioni a livello di sistema operativo, come allocazione e isolamento delle risorse.

```

File: content/it/runtime.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Nel tempo, diversi implementazioni di container runtime hanno continuato ad evolversi, portando alla Specifica OCI, che è diventata lo standard per i runtime dei container.
```
Comment (ugho16) [inline content/it/runtime.md:25]:
```suggestion
Nel tempo, diverse implementazioni di container runtime hanno continuato ad evolversi, portando alla Specifica OCI, che è diventata lo standard per i runtime dei container.
```

File: content/it/runtime.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+È un'[astrazione](/it/abstraction/) del sistema operativo sottostante che traduce i comandi del programma in azioni relative per il sistema operativo.
+
+Nel contesto [cloud native](/it/cloud-native-apps/), per _runtime_ si intende container runtime. 
+Un container runtime è un implementazione che segue le specifiche [Open Container Initiative](https://opencontainers.org/) per garantire una gestione coerente dei container nelle differenti tecnologie di orchestrazione degli stessi.
+
+## Quali problematiche affronta
+Senza l'astrazione di un runtime del container, l'applicazione dovrebbe occuparsi di tutte le meccaniche di ciascun sistema operativo, aumentando la complessità dell'esecuzione dell'app.
+
+## In che modo aiuta
+I container runtime sono un componente necessario degli orchestratori di container come Kubernetes.
```
Comment (ugho16) [inline content/it/runtime.md:20]:
```suggestion
I container runtime sono un componente necessario degli orchestratori di container come [Kubernetes](/it/kubernetes). 
```

Comment (ugho16) [reply content/it/runtime.md:20]: Non c'è nell'originale ma secondo me ci sta bene

**Review by ugho16 — CHANGES_REQUESTED** [review]

File: content/it/runtime.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+title: Runtime
+status: Completed
+category: concetto
+tags: ["applicazione", "", ""]
+---
+
+## Cos'è
+
+Un runtime, in generale, esegue un software.
+È un'[astrazione](/it/abstraction/) del sistema operativo sottostante che traduce i comandi del programma in azioni relative per il sistema operativo.
```
Comment (annalisag-spark) [inline content/it/runtime.md:11]:
```suggestion
È un'[astrazione](/it/abstraction/) del sistema operativo sottostante che traduce i comandi del programma in azioni corrispondenti perché siano utili al sistema operativo stesso.
```

File: content/it/runtime.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Nel contesto [cloud native](/it/cloud-native-apps/), per _runtime_ si intende container runtime.
```
Comment (annalisag-spark) [inline content/it/runtime.md:13]:
```suggestion
Nel contesto [cloud native](/it/cloud-native-apps/), il termine runtime si riferisce generalmente al runtime dei container. 
```

File: content/it/runtime.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+## Cos'è
+
+Un runtime, in generale, esegue un software.
+È un'[astrazione](/it/abstraction/) del sistema operativo sottostante che traduce i comandi del programma in azioni relative per il sistema operativo.
+
+Nel contesto [cloud native](/it/cloud-native-apps/), per _runtime_ si intende container runtime. 
+Un container runtime è un implementazione che segue le specifiche [Open Container Initiative](https://opencontainers.org/) per garantire una gestione coerente dei container nelle differenti tecnologie di orchestrazione degli stessi.
+
+## Quali problematiche affronta
+Senza l'astrazione di un runtime del container, l'applicazione dovrebbe occuparsi di tutte le meccaniche di ciascun sistema operativo, aumentando la complessità dell'esecuzione dell'app.
```
Comment (annalisag-spark) [inline content/it/runtime.md:17]:
```suggestion
Senza l'astrazione di un runtime del container, l'applicazione dovrebbe occuparsi di tutte le meccaniche di ciascun sistema operativo, aumentando la complessità dell'esecuzione dell'applicazione.
```

**Review by annalisag-spark — CHANGES_REQUESTED** [review]

**Review by ugho16 — APPROVED** [review]

**Review by annalisag-spark — APPROVED** [review]

**Pull request merged**